src/Components/Aspire.OracleManagedDataAccess/ConfigurationSchema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
```

Comment:
Is there integration with Logging? If so, can we add the log categories to the ConfigurationSchema.json?

See https://github.com/dotnet/aspire/pull/430 for example.

Reply:
Not for the time being. I believe that if they implement DbDataSource they will provide this feature.

FYI: oracle/dotnet-db-samples#329